articles/quickstart/webapp/nodejs-ea/01-login.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Login
+description: This tutorial demonstrates how to add user login, logout, and profile to a Node.js Express application.
+budicon: 448
+topics:
+  - quickstarts
+  - webapp
+  - login
+  - user profile
+  - logout
+  - nodejs
+contentType: tutorial
+useCase: quickstart
+github:
+  path: 01-Login
+---
+
+<%= include('../../../_includes/_new_app', { hideDownloadSample: true, hideDashboardScreenshot: true }) %>
+
+<%= include('../../../_includes/_callback_url') %>
+
+:::note
+If you are running this project on your local machine, the callback URL you need to whitelist in the **Allowed Callback URLs** field is `http://localhost:3000/callback`.
+:::
+
+<%= include('../../../_includes/_logout_url' }) %>
+
+:::note
+If you are running this project on your local machine, the logout URL you need to whitelist in the **Allowed Logout URLs** field is `http://localhost:3000/logout`.
```

Comment:
```suggestion
If you are running this project on your local machine, your application's logout URL is `http://localhost:3000`
```

Reply:
This is the URL that users come back to after logging out. Application setting field is a bit misleading ... same feedback as the above RE: duplicating steps